python/cuspatial/cuspatial/_lib/cpp/coordinate_transform.pxd

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
```

Comment:
You can move all of these into the `setup.py` file to avoid duplicating them in every file. See:

https://github.com/rapidsai/cudf/blob/branch-0.14/python/cudf/setup.py#L57
https://github.com/rapidsai/cudf/blob/branch-0.14/python/cudf/setup.py#L83-L85